apps/docs/content/components/LogoSuite/index.mdx

```diff
@@ -0,0 +1,152 @@
+      alt=""
+    />
+    <Caption>
+      Arrange the logos so that the sizes, heights, colors, and areas of the
+      separate logos create consistency and balance.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/327b2db7-d337-4788-b0f3-e8c5f80c7af7"
+      alt=""
+    />
+    <Caption>
+      Don’t place logos without considering the spatial balance of the logo
+      suite. Arrange to prevent using more than two rows.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Contextual information
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/4c958008-73bb-4739-81fb-adbbc4541cff"
+      alt=""
+    />
+    <Caption>
+      Do publish the logo suite with context. Introduce the logos and, when
+      applicable, share how the logos have been grouped together.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/7099512c-c0ae-4187-ae4e-7214790d5e18"
+      alt=""
+    />
+    <Caption>
+      Don’t use the logo suite without a heading/description. Consider turning
+      off a visible description if needed.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Color uniformity
+
+<DoDontContainer stacked>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/3245bcd5-4304-4d84-872e-6d30b30427d6"
+      alt=""
+    />
+    <Caption>
+      Ensure that the colors are consistent across the logos to match the logo
+      bar option. Ensure spacing between logos are consistent.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/13340707/a81f3423-b708-45c8-affc-d3d45a4b7cd8"
+      alt=""
+    />
+    <Caption>
+      Don’t use logos with inconsistent colors or don’t match the logo bar
+      option. Don’t use inconsistent spacing.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+### Logo bar appearance
+
+The logo suite offers three appearance options:
+
+- `muted`: The muted option is suitable for displaying 6+ logos and displays logos in a `secondary` color. It allows for two rows of logos, maintaining a clean and organized appearance.
+- `emphasis`: The emphasis option is designed for spotlighting ~5 logos in the `primary` color. It displays logos larger, drawing more attention to the logos.
+- `marquee`: The marquee option provides a flowing logo animation and can accommodate up to 12 logos. It is ideal for scenarios with limited page real estate.
+
+The muted and emphasis options on the logo bar contains an optional divider. This divider can serve as a visual separator. These three options can be used alongside one another when logo suite components are used adjacently.
+
+<div>
+  <img
+    src="https://github.com/primer/brand/assets/13340707/5ad51788-25a4-401a-91b0-02ec7407aa31"
+    alt=""
+  />
+  <Caption>
+    The muted option is the default logo bar. It is the best option for creating
```

Comment:
```suggestion
    The muted option is the default logo bar. To display 6-12 logos in 2 rows, it is preferred to use the muted option instead of emphasis or marquee. 
```